docs/repo/property-pages/property-specification.md

```diff
@@ -285,6 +285,16 @@ a DynamicEnumProperty instead of a StringProperty. All `SupportedValues` that ar
 will be added as unchecked list items.
 
 #### Example
+The xliff namespace can be defined as follows in your Rule definition. 
+An example can be found in [this property page](https://github.com/dotnet/project-system/blob/main/src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/BuildPropertyPage.xaml).
```

Comment:
Why put this comment here? The xliff namespace doesn't actually appear in the example.

Also, translation with xliff-tasks is specific to our repo and a few others. It isn't really a general purpose translation mechanism that we should suggest to anyone writing XAML `Rule` files.

Reply:
XliffTasks is used by every repo that uses [Arcade](https://github.com/dotnet/arcade/blob/3900335a25247d931fc62a62a1ebc30acbb5403d/src/Microsoft.DotNet.Arcade.Sdk/tools/Localization.targets#L26), which is a fairly good number of them in DotNet. They actually have a bunch of automation around localization. Not sure how many of those repos use XAML Rule files, though.